internal/handlers/telegram/documentHandler.go

```diff
@@ -0,0 +1,39 @@
+package telegram
+
+import (
+	tgbotapi "github.com/go-telegram-bot-api/telegram-bot-api"
+)
+
+
+type DocumentObject struct {
+	EnableLink bool
+	Link string
+	Object *tgbotapi.Document
```

Comment:
Instead of calling this Object, we can call this something a little nicer like `Document`.

Reply:
Sure, I like the idea.